scripts/core/context.yml

```diff
@@ -197,6 +210,15 @@ params:
       name: hNativeContext
       desc: |
             [in] the native handle of the context.
+    - type: uint32_t
+      name: numDevices
+      desc: "[in] number of devices associated with the context"
+    - type: "const $x_device_handle_t*"
+      name: phDevices
+      desc: "[in][range(0, numDevices)] list of devices associated with the context"
+    - type: "const $x_context_native_desc_t*"
+      name: pContextNativeDesc
```

Comment:
In other creation entry-points, and in relation to using the `*_properties_t` suffix, this argument should be called `pProperties`.